### PR TITLE
Feat/improve UI faucet

### LINKED ISSUE #1258: Testnet - Issue Tokens
For the faucet of Libre tokens for Testnet it would be more intuitive to have 'Account Name' instead of 'Account (Libre 500)' written in 'Issue Tokens' box

Also when setting account it is not clear the parameters of 'account name' such as 'no capital letters'

### PR DESCRIPTION
**Pull request by Torresmorah**
### What does this PR do?

- Resolve #1258 

### Steps to test

1. Run the project with Libre testnet configuration
2. Go to faucet page
3. Check the forms and the texts

### Screenshots

imagen
imagen

### CODE REVIEW

File: webapp/src/language/en.json [hunk 1]
```diff
@@ -197,11 +197,15 @@
     "createAccount": "Create Faucet Account",
     "newCreatedAccount": "Your new account is",
     "transferTokensTransaction": "Tokens transferred correctly. Check it on explorer here",
```
Comment (xavier506) [inline webapp/src/language/en.json:199]: does this text have a link?

Comment (Torresmorah) [reply webapp/src/language/en.json:199]: Yes, when the faucet is performed, it show a message with the link

imagen

https://github.com/edenia/antelope-tools/blob/35efac196af18167def5bd6ac8ade1e3929daf49/webapp/src/routes/Faucet/index.js#L129-L134

File: webapp/src/language/en.json [hunk 2]
```diff
@@ -197,11 +197,15 @@
     "createAccount": "Create Faucet Account",
```
[the remark this answers is not in the document]
Comment (Torresmorah) [reply webapp/src/language/en.json]: This text is from the account creator form

imagen

File: webapp/src/language/en.json (hunk 1 above)
Comment (xavier506) [reply webapp/src/language/en.json:199]: OK lets use this JSON for the english texts please: 
```
"faucetRoute": {
    "createAccount": "Create Your Blockchain Account",
    "newCreatedAccount": "Congratulations! Your new account has been created:",
    "transferTokensTransaction": "Tokens have been successfully transferred. View the transaction on the explorer:",
    "createButton": "Create Your Account",
    "issueTokens": "Distribute Tokens",
    "getTokens": "Receive Tokens",
    "publicKey": "Enter Public Key (Active/Owner)",
    "accountName": "Choose an Account Name",
    "invalidAccount": "Enter a valid account name, please.",
    "emptyFields": "Complete all fields to proceed with account creation.",
    "accountFormat": "Account name should be 12 characters or fewer. Use lowercase letters (a-z) and numbers (1-5) only.",
    "keyFormat": "Enter a valid ECC key. You can generate one using cleos or any compatible wallet."
}
```

File: webapp/src/language/es.json [hunk 3]
```diff
@@ -200,14 +200,18 @@
     "bpjsonInconsistency": "El bp.json on chain y en of chain no coincide revisalo y realiza los cambios necesarios"
   },
   "faucetRoute": {
-    "createAccount": "Crear Cuenta Faucet",
+    "createAccount": "Crear una Nueva Cuenta",
```
Comment (xavier506) [inline webapp/src/language/es.json:203]: Please use this json for spanish texts :

```
"faucetRoute": {
    "createAccount": "Cree su Cuenta de Blockchain",
    "newCreatedAccount": "¡Felicidades! Su nueva cuenta ha sido creada:",
    "transferTokensTransaction": "Los tokens han sido transferidos exitosamente. Vea la transacción en el explorador:",
    "createButton": "Cree su Cuenta",
    "issueTokens": "Distribuya Tokens",
    "getTokens": "Reciba Tokens",
    "publicKey": "Introduzca la Clave Pública (Activa/Propietario)",
    "accountName": "Elija un Nombre de Cuenta",
    "invalidAccount": "Por favor, introduzca un nombre de cuenta válido.",
    "emptyFields": "Complete todos los campos para continuar con la creación de la cuenta.",
    "accountFormat": "El nombre de la cuenta debe tener 12 caracteres o menos. Utilice solo letras minúsculas (a-z) y números (1-5).",
    "keyFormat": "Introduzca una clave ECC válida. Puede generar una usando cleos o cualquier billetera compatible."
}
```

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**